app/views/npq_separation/admin/users/change_trn/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= govuk_back_link href: npq_separation_admin_user_path(@user) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
```

Comment:
I made the width full here, because even at three-quarters, the Participant ID wraps.